ci/deploy.sh

```diff
@@ -198,6 +214,10 @@ EOF
         install -Dsm755 -t pkg/debian/usr/bin target/release/strip-ansi-escapes
 
         deps=$(cd pkg && dpkg-shlibdeps -O -e debian/usr/bin/*)
+	      mv pkg/debian/postinst pkg/debian/DEBIAN/postinst
+        chmod 0555 pkg/debian/DEBIAN/postinst
```

Comment:
These `0555` should probably be `0755`?  `555` is a bit unusual!  Does the deb stuff require it?

Reply:
Docs says something like 0555 to 0755. I don't think it matters. But we can make it 755 too.